packages/ui-react-rx/src/ApiObservable.ts

```diff
@@ -17,54 +19,48 @@ type OptDate = Date | undefined;
 
 export default class ObservableApi implements ObservableApiInterface {
   private api: RxApiInterface;
-  private lastBestNumber: OptBN;
 
   constructor (api: RxApiInterface) {
     this.api = api;
-
-    this.api.chain
-      .getHead()
-      .pipe(
-        switchMap((hash: Uint8Array) =>
-          this.api.chain.getHeader(hash)
-        ),
-        map((header?: Header) =>
-          header && header.number
-        )
-      )
-      .toPromise()
-      .catch(() => undefined)
-      .then((bestNumber?: BN) => {
-        this.lastBestNumber = this.lastBestNumber || bestNumber;
-
-        console.error('this.lastBestNumber', this.lastBestNumber);
-      });
   }
 
   private combine <T, R> (observables: Array<Observable<any>>, mapfn: (combined: R) => T): Observable<T> {
     return combineLatest(...observables).pipe(map(mapfn));
   }
 
+  // FIXME This should not be needed when we have the auto-update storage entries, well, at
```

Comment:
This to be fixed before the PR is ready. New master supports this, as does the 0.2 backports.